package.json

```diff
@@ -1,7 +1,10 @@
 {
   "name": "font-cacher",
   "version": "1.1.0",
-  "main": "index.js",
+  "main": "src/index.js",
+  "scripts": {
+    "test": "node tests/index.js"
```

Comment:
Especially if you're actively supporting tests here, let jest actually fail the call.